apps/server/src/services/runtime-service/RuntimeService.ts

```diff
@@ -93,6 +93,25 @@ class RuntimeService {
       const rundown = getPlayableEvents();
       this.eventTimer.roll(rundown);
     }
+
+    const timerPhaseChanged = RuntimeService.previousState.timer.phase !== newState.timer.phase;
```

Comment:
On the first time the app runs, `RuntimeService.previousState` is an empty object

we need to guard this check otherwise the app will crash on the first start
something like

```suggestion
    const timerPhaseChanged = RuntimeService.previousState.timer?.phase !== newState.timer.phase;
```

You can recreate this by navigating to the user install folder (for me in macos `/Users/carlos/Library/Application Support/Ontime/`) and deleting the ontime.restore file

On crash a crash log is made in the same folder, mine contains

```ts
  "error": {
    "message": "Cannot read properties of undefined (reading 'phase')",
    "stack": 

"TypeError: 
    Cannot read properties of undefined (reading 'phase')\n    
    at RuntimeService.checkTimerUpdate (file:///Users/carlos/ontime/apps/server/src/services/runtime-service/RuntimeService.ts:107:66)\n    <---------
    at RuntimeService.descriptor.value (file:///Users/carlos/ontime/apps/server/src/services/runtime-service/RuntimeService.ts:554:35)\n   
    at TimerService.onUpdateCallback (file:///Users/carlos/ontime/apps/server/src/services/runtime-service/RuntimeService.ts:134:48)\n    
    at TimerService.update (file:///Users/carlos/ontime/apps/server/src/services/TimerService.ts:92:10)\n   
    at Timeout._onTimeout (file:///Users/carlos/ontime/apps/server/src/services/TimerService.ts:35:12)\n   
    at listOnTimeout (node:internal/timers:569:17)\n    
    at processTimers (node:internal/timers:512:7)"
  },
```

Reply:
Good catch I trusted the typing too much 😂